### DISCUSSION TITLE
2006/10/25/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-4

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#124] [Announcements]
# 2006/10/25/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-4

上午工作时，不经意间线右侧的窗外望了一下，一部手臂长长的吊车映入眼帘。这让我突然想起的昨晚的梦，梦中也有一部吊车。我看着它晃来晃去，突然就坍塌了。关于这个梦就只能想起这么一点了。

https://blog.qizhen.xyz/2006/10/25/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-4

**Comment by ruanqizhen** [comment]
之前博客下的留言：

¨Randy＂﹏。
2006年10月26日 19:26 编辑
日有所思夜有所梦!
白天晚上有什么不一样噢?

回复
Mavis
2006年10月25日 23:04 编辑
看来明天该穿长袖了

回复
竹贤
2006年10月25日 22:06 编辑
入秋后，天要渐渐凉了，注意别受凉。晚间可以盖一条薄薄的被子了。
